src/compute/src/sink/persist_sink.rs

```diff
@@ -209,16 +212,18 @@ where
         &persist_errs,
         Arc::clone(&persist_clients),
         compute_state.read_only_rx.clone(),
+        obey_read_only_mode,
```

Comment:
If you wanted to avoid the extra param you could make the previous one an `Option` and pass `None` if the dyncfg is false. But it's fine not to overcomplicate this, since we're going to rip it out soon again, hopefully!